prowler/providers/aws/services/securityhub/securityhub_service.py

```diff
@@ -28,7 +28,7 @@ def _describe_hub(self, regional_client):
                     self.securityhubs.append(
                         SecurityHubHub(
                             arn=self.audited_account_arn,
```

Comment:
Here the ARN has not been modified and it should be `arn:aws:securityhub:us-east-1:123456789012:hub/unknown`

Reply:
Done